lib/widgets/drive.coffee

```diff
@@ -34,6 +38,17 @@ getDrives = ->
 			async.reject drives, drivelist.isSystem, (results) ->
 				return callback(null, results)
 
+cleanupList = (previous,currrent) ->
+	removedDrive =
+		name: "#{currrent.device} (#{currrent.size}) - #{currrent.description}"
+		value: currrent.device
+	return _.reject previous, (drive) ->
+		# The reason we use _.pick and pass a custom object in _.isEqual is due to the fact the initial objects (previous, current)
```

Comment:
This comment is amazing! For readability purposes, we add a blank line above each comment.